clusterman/batch/node_migration.py

```diff
@@ -97,7 +151,7 @@ def run(self):
             if "event_queue" in config["trigger"]:
                 event_queues.add(config["trigger"]["event_queue"])
```

Comment:
Currently we store all SQS URLs in common configuration file (clusterman.yaml)
Should we move this URL as well?

Reply:
I thought it could come handy to have the option of setting it individually per-pool, but it's true that passing cluster and pool inside the events themselves kind of nullifies the utility of having the option. I'll rework things a little.